src/cmo.adoc

```diff
@@ -931,10 +931,13 @@ Encoding::
 Description::
 
 A *cbo.flush* instruction performs a flush operation on the cache block whose
-effective address is the base address specified in _rs1_. The offset operand may
-be omitted; otherwise, any expression that computes the offset shall evaluate to
-zero. The instruction operates on the set of coherent caches accessed by the
-agent executing the instruction.
+that contains the address specified in _rs1_. It is not required that _rs1_ is
+aligned to the size of a cache block. On faults the address stored in the *tval
+CSR is _rs1_; not the base address of the cache block. The instruction operates
+on the set of coherent caches accessed by the agent executing the instruction.
+
+The assembly _offset_ operand may be omitted. If it isn't then any expression
+that computes the offset shall evaluate to zero.
```

Comment:
This doesn't belong in the ISA spec; delete it.

Reply:
Ah, I see this has been factored out of the previous paragraph and isn't a new addition.  I still think it doesn't belong here, but since it's better to keep the topic of the PR restricted, we should leave it alone.